pkg/cmd/util.go

```diff
@@ -24,6 +24,8 @@ func AddCommonFlags(c *cobra.Command, cfg *api.Config) {
 		"DEPRECATED: Always pull the builder image even if it is present locally")
 	c.Flags().VarP(&(cfg.BuilderPullPolicy), "pull-policy", "p",
 		"Specify when to pull the builder image (always, never or if-not-present)")
+	c.Flags().Var(&(cfg.PreviousImagePullPolicy), "incremental-pull-policy",
+		"Specify when to pull the previous image for incremental builds (always, never or if-not-present)")
```

Comment:
As discussed on IRC: we'll leave previous image pull policy `always` in the main part of s2i, this way always will be the default for OpenShift usecase, which is embedding the core of s2i. Whereas in `main.go` we'll force previous image pull policy to be `pull-if-not-present`, this will address the CLI usecase. @bparees can you apply those changes?